pkg/apis/pipeline/v1beta1/resultref.go

```diff
@@ -23,6 +23,7 @@ import (
 )
 
 // ResultRef is a type that represents a reference to a task run result
+// +k8s:openapi-gen=false
```

Comment:
what's the reason for this?

Reply:
`ResultRef` isn't actually used anywhere, so far as I can tell. Being me with my tendency to not muck around with things I don't actually have full context on, I decided that it didn't make sense to care about whether `ResultRef`'s fields were properly tagged if it's never used but didn't want to go too far and delete the type completely. So I opted to exclude `ResultRef` from the OpenAPI stuff entirely, hence the annotation.

Reply:
it looks like it's used here https://github.com/tektoncd/pipeline/blob/main/pkg/reconciler/pipelinerun/resources/resultrefresolution.go so I would include it in codegen

Reply:
It doesn't look like this comment was addressed

Reply:
Ah, don't know how I missed both the usage and your comment! 🤦 Fixing now!